symfony/translation/3.3/config/packages/translation.yaml

```diff
@@ -1,7 +1,8 @@
 framework:
     default_locale: '%locale%'
     translator:
-        paths:
-            - '%kernel.project_dir%/translations'
+        # additional translation paths
```

Comment:
maybe we should remove this one? I'm not sure it's that common to add more paths, so better keep the config simpler? (same below of course)

Reply:
I think so too, but there is no consensus about that (others member think otherwise) at least for Twig `paths`. See previous comments.